src/Swup.ts

```diff
@@ -45,6 +41,7 @@ export type Options = {
 	skipPopStateHandling: (event: any) => boolean;
 	ignoreVisit: (url: string, { el, event }: { el?: Element; event?: Event }) => boolean;
 	resolveUrl: (url: string) => string;
+	transitionRoot: boolean;
```

Comment:
I think instead of a boolean I would prefer a more descriptive `"html" | "containers"` here

Reply:
Renamed to `animationScope: 'html' | 'containers'`